### PR TITLE
metrics: update hot scheduler metrics desc

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

update hot scheduler metrics desc, add r/w information.

### Check List

Tests

- Unit test

### CODE REVIEW

File: server/schedulers/metrics.go [hunk 1]
```diff
@@ -87,6 +87,14 @@ var balanceDirectionCounter = prometheus.NewCounterVec(
 		Help:      "Counter of direction of balance related schedulers.",
 	}, []string{"type", "source", "target"})
 
+var hotDirectionCounter = prometheus.NewCounterVec(
+	prometheus.CounterOpts{
+		Namespace: "pd",
+		Subsystem: "scheduler",
+		Name:      "hot_region_direction",
+		Help:      "Counter of hot region scheduler.",
+	}, []string{"type", "rwType", "store", "direction"})
```
Comment (disksing) [inline server/schedulers/metrics.go:96]: rwType -> rw_type?

Comment (disksing) [reply server/schedulers/metrics.go:96]: or just rw

**Comment by lhy1024** [comment]
image
image
image

**Review by Yisaer — APPROVED** [review]
LGTM

**Comment by rleungx** [comment]
The metrics of transfer leader and move peer are not consistent. e.g. `in-1` and `move-peer-2`

**Comment by lhy1024** [comment]
> The metrics of transfer leader and move peer are not consistent. e.g. `in-1` and `move-peer-2`

Fixed

**Review by disksing — APPROVED** [review]

**Pull request merged**